fe/src/main/java/org/apache/doris/load/loadv2/LoadManager.java

```diff
@@ -602,6 +602,9 @@ public void readFields(DataInput in) throws IOException {
                 map.put(loadJob.getLabel(), jobs);
             }
             jobs.add(loadJob);
+            if (!loadJob.isCompleted()) {
```

Comment:
Add some comment to explain this operation

Reply:
OK

Reply:
I think you should use another way to achieve this. When this function is called, Catalog is not ready. However you start to use Catalog's item, this is error-prone.

Reply:
The callback need to be add in here. The reason is that the replay of txn maybe use the callback and reload the job state.